src/code/z_play.c

```diff
@@ -1027,17 +1029,16 @@ void Play_UpdateMain(PlayState* this) {
         }
     }
 
-    if (!sp5C || gDbgCamEnabled) {
-        s32 sp54;   // camId
-        Vec3s sp48; // InputDir
+    if ((!sp5C) || gDbgCamEnabled) {
```

Comment:
```suggestion
    if (!sp5C || gDbgCamEnabled) {
```
Here and other places, a lot of extra brackets seems to have been added